code/model/modelread.cpp

```diff
@@ -1700,13 +1700,46 @@ int read_model_file(polymodel * pm, const char *filename, int n_subsystems, mode
 						Error( LOCATION, "Model '%s' is chunked.  See John or Adam!\n", pm->filename );
 					}
 				}
-				pm->submodel[n].bsp_data_size = cfread_int(fp);
-				if ( pm->submodel[n].bsp_data_size > 0 )	{
-					pm->submodel[n].bsp_data = (ubyte *)vm_malloc(pm->submodel[n].bsp_data_size);
-					cfread(pm->submodel[n].bsp_data,1,pm->submodel[n].bsp_data_size,fp);
-					swap_bsp_data( pm, pm->submodel[n].bsp_data );
-				} else {
-					pm->submodel[n].bsp_data = NULL;
+				//ShivanSpS - if pof version is 2118 or higher load bsp_data as it is, otherwise, align it
+				if (pm->version >= 2118)
+				{
+					pm->submodel[n].bsp_data_size = cfread_int(fp);
+					if (pm->submodel[n].bsp_data_size > 0) {
+						pm->submodel[n].bsp_data = (ubyte*)vm_malloc(pm->submodel[n].bsp_data_size);
+						cfread(pm->submodel[n].bsp_data, 1, pm->submodel[n].bsp_data_size, fp);
+						swap_bsp_data(pm, pm->submodel[n].bsp_data);
+					}
+					else {
+						pm->submodel[n].bsp_data = nullptr;
+					}
+				}
+				else
+				{
+					pm->submodel[n].bsp_data_size = cfread_int(fp);
+					if (pm->submodel[n].bsp_data_size > 0) {
+						mprintf(("BSP_Data is being aligned.\n"));
```

Comment:
This will log print will only spam the log file and is not really useful for mod debugging.

Reply:
Ok ill do that